jax/numpy/lax_numpy.py

```diff
@@ -3150,6 +3151,18 @@ def sort(a, axis=-1, kind='quicksort', order=None):
   else:
     return lax.sort(a, dimension=_canonicalize_axis(axis, ndim(a)))
 
+@_wraps(np.sort_complex)
+def sort_complex(a):
+    a.sort()
```

Comment:
One other thing: JAX arrays are immutable, so they don't support in-place sorts.  This would need to be `a = sort(a)` for the algorithm to work correctly.